.github/workflows/MainDistributionPipeline.yml

```diff
@@ -14,19 +14,19 @@ concurrency:
 jobs:
   duckdb-stable-build:
     name: Build extension binaries
-    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@v1.2.1
+    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@v1.2.2
```

Comment:
There is no 1.2.2 branch there (yet?)

I would keep this to `main`

Reply:
To be more clear, ci_tools_versions is always `main` (in all 4 places)

Reply:
I think we should add the v1.2.2 branch there before merging this